src/components/TaskSummary/index.js

```diff
@@ -2,11 +2,19 @@ import React, { Fragment } from 'react'
 import i18n from '@dhis2/d2-i18n'
 import { eventEmitter } from '../../services'
 import { Loading } from '../'
-import { Totals, TypeStats, Conflicts, Messages, Summaries } from './helpers'
+import {
+    Errors,
+    Totals,
+    TypeStats,
+    Conflicts,
+    Messages,
+    Summaries,
+} from './helpers'
 import s from './styles.module.css'
 
 const initialState = {
     loading: false,
+    errors: '',
```

Comment:
It looks like `errors` (a string) and `error` (an array) is mixed up through out.

Reply:
Thanks